config/locales/views/home/home.yml

```diff
@@ -53,7 +53,19 @@ en:
         header: Empower pupils
         subtext: Number of participating schools using Energy Sparks with their eco-team to inform impactful action
     sub_title: A unique school-specific energy management tool and education programme
+    testimonials:
```

Comment:
Aside: we'll want to move these to some content management eventually. Need to decide whether to add that once the homepage is done or after implementing the other pages.

Reply:
You were talking about moving the yaml to the database a bit ago. Is this what you mean?

Reply:
No, I mean that we should have a Testimonials model (or similar) with admin forms so they can be managed in the app, as we do with Case Studies. The model would sync to transifex.